cmd/kube-apiserver/app/options/validation.go

```diff
@@ -126,6 +127,10 @@ func validateTokenRequest(options *ServerRunOptions) []error {
 	return errs
 }
 
+func validateShowHiddenMetricsVersion(options *ServerRunOptions) []error {
+	return metrics.ValidateShowHiddenMetricsVersion(options.ShowHiddenMetricsForVersion)
+}
+
```

Comment:
why the wrapper?

Reply:
Share validation logic with other kubernetes binaries. Is it ok?

Reply:
why not just call `metrics.ValidateShowHiddenMetricsVersion` directly below?